### PR TITLE
Feature 1731 authorship

### LINKED ISSUE #1731: Update the User's Guide for the 10.0.0 release.
## Describe the Task ##
Prior to the actual 10.0.0 release, we'll need to make some updates to the User's Guide. Please list the changes that need to be made here:
- Revise and rotate the User's Guide authorship list, as directed by @TaraJensen 
- Add 10.0.0 release notes (this will be done as part of the release process)
- Create a new 10.0.0 flowchart and drop it into the docs.
- Review/update section 1.4 "MET components" as needed.

Please note additional required changes here.

### Time Estimate ###
4 hours

### Sub-Issues ###
Consider breaking the task down into sub-issues.
No sub-issues needed.

### Relevant Deadlines ###
Prior to 10.0.0 release.

### Funding Source ###
2702691

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required: John HG and Julie
- [x] Select **scientist(s)** or **no scientist** required: John Opatz to review/edit changes

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Review **projects** and select relevant **Repository** and **Organization** ones or add "alert:NEED PROJECT ASSIGNMENT" label
- [x] Select **milestone** to next major version milestone or "Future Versions"

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdatadb](https://github.com/dtcenter/METdatadb/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)
No impacts

## Task Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)**, **Project(s)**, **Milestone**, and **Linked issues**
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by JohnHalleyGotway**
## Pull Request Testing ##

These are the same authorship list changes that were made for METplus:
https://github.com/dtcenter/METplus/pull/912

- [x] Describe testing already performed for these changes:</br>
Only updating the docs.

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
Please review authorship changes to docs at:
https://met.readthedocs.io/en/feature_1731_docs/index.html

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**

- [x] Do these changes include sufficient testing updates? **[Yes]**
None needed.

- [x] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Please complete this pull request review by **[May 4th or 5th]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**, **Project(s)**, and **Milestone**
- [x] After submitting the PR, select **Linked Issues** with the original issue number.
- [x] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [x] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: met/docs/Users_Guide/reformat_point.rst [hunk 1]
```diff
@@ -837,7 +836,145 @@ We will not give a detailed description of each CALIPSO data product that lidar2
...
+_________________
+
+
+.. code-block:: none
+
+		message_type           = [];
+		message_type_group_map = [];
+		message_type_map       = [];
+		station_id             = [];
+      elevation_range        = { ... };
```
Comment (jprestop) [inline met/docs/Users_Guide/reformat_point.rst]: I see that elevation_range, level_range, and obs_var are not lined up with the other vars.  Is that intended?

File: met/docs/Users_Guide/reformat_point.rst [hunk 2]
```diff
@@ -837,7 +836,145 @@ We will not give a detailed description of each CALIPSO data product that lidar2
...
+ioda2nc configuration file
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The default configuration file for the IODA2NC tool named **IODA2NcConfig_default** can be found in the installed *share/met/config* directory. It is recommended that users make a copy of this file prior to modifying its contents.
+
+The IODA2NC configuration file is optional and only necessary when defining filtering the input observations or defining time summaries. The contents of the default IODA2NC configuration file are described below.
+
+__________________
+
+.. code-block:: none
```
Comment (jprestop) [inline met/docs/Users_Guide/reformat_point.rst:916]: I see there is a line above the code blocks.  In the ".. code-block:: none" statements, could you please try changing those to be ".. code-block:: ini" and see if that gets rid of the line?  If not, it's fine and I'll look into what's going on.

File: met/docs/Users_Guide/reformat_point.rst [hunk 3]
```diff
@@ -837,7 +836,145 @@ We will not give a detailed description of each CALIPSO data product that lidar2
...
+
+
+ioda2nc configuration file
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The default configuration file for the IODA2NC tool named **IODA2NcConfig_default** can be found in the installed *share/met/config* directory. It is recommended that users make a copy of this file prior to modifying its contents.
+
+The IODA2NC configuration file is optional and only necessary when defining filtering the input observations or defining time summaries. The contents of the default IODA2NC configuration file are described below.
+
+__________________
```
Comment (jprestop) [inline met/docs/Users_Guide/reformat_point.rst:914]: Oh wait, maybe this line is the cause of the line and maybe it is intended?

File: met/docs/Users_Guide/reformat_point.rst (hunk 1 above)
Comment (JohnHalleyGotway) [reply met/docs/Users_Guide/reformat_point.rst]: Silly tabs vs spaces. I will fix this to line them up and push an update.

File: met/docs/Users_Guide/reformat_point.rst (hunk 2 above)
Comment (JohnHalleyGotway) [reply met/docs/Users_Guide/reformat_point.rst:916]: The line above this code block looks correct to me. We use that horizontal line between the config file entries being described. It matches the formatting used for the other point pre-processing tools, as well as config entries in Point-Stat (for example):
https://met.readthedocs.io/en/feature_1731_docs/Users_Guide/point-stat.html#point-stat-configuration-file

File: met/docs/Users_Guide/reformat_point.rst (hunk 3 above)
Comment (JohnHalleyGotway) [reply met/docs/Users_Guide/reformat_point.rst:914]: Yes, it's intended.

File: met/docs/Users_Guide/reformat_point.rst (hunk 2 above)
Comment (jprestop) [reply met/docs/Users_Guide/reformat_point.rst:916]: Thanks for letting me know.  I did not realize that.

**Review by jprestop — APPROVED** [review]
I reviewed the changes, and I approve this request.

**Pull request merged**